bundles/org.openhab.ui.basic/web-src/_theming.scss

```diff
@@ -7,6 +7,8 @@ body[data-theme="default"] {
 	--container-text-color: #616161;
 	--switch-on-track-bg: #9fa8da;
 	--border-color: #ccc;
+	--input-border-color: rgba(0,0,0,.12);
```

Comment:
Couldn't we use an existing variable for border color ? Maybe "--border-color" (to be checked where it is used) ?

Reply:
--border-color will do. It is used for the lines between the rows, but it does not harm if the underline in the input field has the same color.

Reply:
Ok, let's do it as you propose.